### PR TITLE
Added tests for Person, BaseEntity, and NamedEntity

### PR DESCRIPTION
**Pull request by abhandal**
Added tests for BaseEntity, Person, and NamedEntity as they are DAOs (Data Access Objects) which perform database manipulations on core data.

### CODE REVIEW

File: src/test/java/org/springframework/samples/petclinic/model/PersonTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    @Before
+    public void setUp(){
+        personObj = new Person();
+    }
+
+    // Testing the DAO of Person class by verifying the setter and getter of FirstName
+    @Test
+    public void testSetAndGetFirstName() {
+        personObj.setFirstName("Johnny");
```
Comment (amawai) [inline src/test/java/org/springframework/samples/petclinic/model/PersonTest.java:20]: Is the `getFirstName` supposed to be getting tested here?

**Review by amawai — CHANGES_REQUESTED** [review]
Good test coverage, especially since these classes form the basis of other frequently used classes, and would require testing.
Just one small issue, it appears as though `getFirstName` is not tested for the `Person` class

**Review by jadmalek — APPROVED** [review]
In the same vain of the Owner, Pet and Visit classes tests, the core functionality of these classes are being tested and are thus sufficient as far as coverage is concerned. Could have been more consistent with the naming but otherwise, good job.

File: src/test/java/org/springframework/samples/petclinic/model/PersonTest.java (hunk 1 above)
Comment (abhandal) [reply src/test/java/org/springframework/samples/petclinic/model/PersonTest.java:20]: Added the assertEqual, must have forgotten it.

**Review by amawai — APPROVED** [review]

**Comment by jadmalek** [comment]
Addresses #6.

**Pull request merged**